### DISCUSSION TITLE
Offline Mode?

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [photoprism/photoprism#2969] [General]
Hello!

I want to migrate about 30GB+ of old images, videos and such to Photoprism, also as a way to finally clean up my whole collection of data.

I saw that there is an installable PWA - however, do I always have to be online to use it or can I pick certain albums and make them available offline as well? This would be specifically neat for bringing them to a family event where I know the internet is often quite bad.

Thanks and kind regards! =)

**Comment by graciousgrey** [comment]
Hello :)

Unfortunately that's not possible at the moment. Feel free to submit a feature request for this!
